src/features/views/layout/SingleViewLayout.tsx

```diff
@@ -190,6 +194,46 @@ const SingleViewLayout: FunctionComponent<SingleViewLayoutProps> = ({
         defaultTab="/"
         ellipsisMenuItems={ellipsisMenu}
         fixedHeight={true}
+        subtitle={
+          // TODO: Replace with model eventually
+          <ZUIQuery
+            queries={{
+              colsQuery: useQuery(
+                ['view', viewId, 'columns'],
+                getViewColumns(orgId as string, viewId as string)
+              ),
+              rowsQuery: useQuery(
+                ['view', viewId, 'rows'],
+                getViewRows(orgId as string, viewId as string)
+              ),
+            }}
+          >
+            {({ queries: { colsQuery, rowsQuery } }) => (
+              <ZUIIconLabelRow
+                iconLabels={[
+                  {
+                    icon: <Group />,
+                    label: (
+                      <FormattedMessage
+                        id="pages.people.views.layout.subtitle.people"
+                        values={{ count: rowsQuery.data.length }}
+                      />
+                    ),
+                  },
+                  {
+                    icon: <ViewColumnOutlined />,
+                    label: (
```

Comment:
Is there a specific reason here to use the `FormattedMessage` component rather than the `intl.formatMessage` function? Since this desicion made you have to change the type that `label` takes, I thought it might make a difference somehow which of them you use here?

Reply:
I generally like `<FormattedMessage>` better, because it's less syntactically hysterical ({:''}). 😊 

The `intl.formatMessage({ id: 'some.string' })` option always returns a string, and I think `<FormattedMessage/>` does so nowadays as well, but it used to return a span. So I only use it where a span is acceptable.

But I've been thinking about this recently, and I think I'd like us to create a new cook to wrap `useIntl()` and return a less verbose method, like this: 

```tsx
const intl = useZetkinIntl();

// Shorter name, no object as first param
intl.msg('id.without.object', { someValue: 1, otherValue: 2 });
```

Until we do, I think I will keep shying away from `intl.formatMessage()` whenever I can, unless anyone objects. 😊

Reply:
Thank you for good explanation! I like the idea of wrapping it!